pkg/sdk/poc/generator/templates/sub_templates/implementation_functions.tmpl

```diff
@@ -13,9 +14,21 @@
         return resultList, nil
         }
     {{ else if eq .Name "ShowByID" }}
+        {{ $idkind := .ObjectInterface.ObjectIdentifierKind }}
         func (v *{{ $impl }}) ShowByID(ctx context.Context, id {{ .ObjectInterface.IdentifierKind }}) (*{{ .ObjectInterface.NameSingular }}, error) {
-        // TODO: adjust request if e.g. LIKE is supported for the resource
-        {{ $impl }}, err := v.Show(ctx, NewShow{{ .ObjectInterface.NameSingular }}Request())
+            request := NewShow{{ .ObjectInterface.NameSingular }}Request()
+            {{- if not .ShowByIDFiltering }}
+                // TODO: adjust request if e.g. LIKE is supported for the resource
```

Comment:
We can leave the default behavior as it is, but we can point the SDK users to use `	g.ShowByIDLikeFiltering,` and others more explicitly.

Reply:
How about: // TODO: adjust request  if e.g. LIKE is supported for the resource with `ShowByIDLikeFiltering` and other filters for "ShowByIdOperation" in the definition file."

Reply:
Sounds good.

Reply:
I rephrased it a bit:
// TODO: adjust request with resource supported filtering e.g. for LIKE use 'ShowByIDLikeFiltering' in 'ShowByIdOperation()' inside the definition file.